extensions/amp-accordion/1.0/accordion.js

```diff
@@ -61,6 +61,7 @@ function AccordionWithRef(
   ref
 ) {
   const [expandedMap, setExpandedMap] = useState(EMPTY_EXPANDED_MAP);
+  const eventMap = useRef(/** @type {!Object} */ ({}));
```

Comment:
Nit: let's call this `eventMapRef`. And since you now always reconstruct the object, feel free to use some sort of `EMPTY_EVENT_MAP` const.

Reply:
Thanks updated in the latest!